src/DB/exerciseData.json

```diff
@@ -249,4 +249,14 @@
     "gh-name": "haldaranup",
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2021/10/t-bar-row-muscles.gif"
   }
+  {
+    "exercise": "Exercise Name",
```

Comment:
And please add the actual name of the exercise like `"exercise": "Heel touches",`